src/stream/frame/content.rs

```diff
@@ -306,6 +306,17 @@ impl<W: io::Write> Encoder<W> {
         Ok(())
     }
 
+    fn involved_people_list(&mut self, content: &InvolvedPeopleList) -> crate::Result<()> {
+        self.encoding()?;
+        for item in &content.items {
+            self.string(&item.involvement)?;
+            self.delim()?;
+            self.string(&item.involvee)?;
+            self.delim()?;
```

Comment:
Should the final item in the list also have a trailing delimiter? Or should it only place the delimiter between strings?

Reply:
Hmm, here's what picard writes when I print `data` as hex in the tests:

```
---- tag::tests::test_ipls_id3v24_utf8 stdout ----
[70, 72, 6f, 64, 75, 63, 65, 72, 00, 44, 61, 76, 65, 20, 55, 73, 68, 65, 72]
[64, 6f, 75, 62, 6c, 65, 20, 62, 61, 73, 73, 00, 49, 73, 72, 61, 65, 6c, 20, 43, 72, 6f, 73, 62, 79, 00, 64, 72, 75, 6d, 73, 20, 28, 64, 72, 75, 6d, 20, 73, 65, 74, 29, 00, 56, 65, 72, 6e, 65, 6c, 6c, 20, 46, 6f, 75, 72, 6e, 69, 65, 72, 00, 70, 69, 61, 6e, 6f, 00, 41, 68, 6d, 61, 64, 20, 4a, 61, 6d, 61, 6c]

---- tag::tests::test_ipls_id3v23_utf16 stdout ----
[ff, fe, 64, 00, 6f, 00, 75, 00, 62, 00, 6c, 00, 65, 00, 20, 00, 62, 00, 61, 00, 73, 00, 73, 00, 00, 00, ff, fe, 49, 00, 73, 00, 72, 00, 61, 00, 65, 00, 6c, 00, 20, 00, 43, 00, 72, 00, 6f, 00, 73, 00, 62, 00, 79, 00, 00, 00, ff, fe, 64, 00, 72, 00, 75, 00, 6d, 00, 73, 00, 20, 00, 28, 00, 64, 00, 72, 00, 75, 00, 6d, 00, 20, 00, 73, 00, 65, 00, 74, 00, 29, 00, 00, 00, ff, fe, 56, 00, 65, 00, 72, 00, 6e, 00, 65, 00, 6c, 00, 6c, 00, 20, 00, 46, 00, 6f, 00, 75, 00, 72, 00, 6e, 00, 69, 00, 65, 00, 72, 00, 00, 00, ff, fe, 70, 00, 69, 00, 61, 00, 6e, 00, 6f, 00, 00, 00, ff, fe, 41, 00, 68, 00, 6d, 00, 61, 00, 64, 00, 20, 00, 4a, 00, 61, 00, 6d, 00, 61, 00, 6c, 00, 00, 00, ff, fe, 70, 00, 72, 00, 6f, 00, 64, 00, 75, 00, 63, 00, 65, 00, 72, 00, 00, 00, ff, fe, 44, 00, 61, 00, 76, 00, 65, 00, 20, 00, 55, 00, 73, 00, 68, 00, 65, 00, 72, 00]
```

So it looks like there is no trailing delimiter other than the closing one.

Reply:
I extended to tests to include writing, and apparently there is some bug. I need to look deeper into it when I find the time.

Reply:
alright, fixed in 2a95bb65df4bf1a751929860e23d9356e8947b2b